modules/orgs/client/views/list.orgs.directive.html

```diff
@@ -27,14 +28,14 @@ <h3>Companies</h3>
     <div class="table-responsive">
       <table class="table table-borderless table-hover company-list">
         <tbody>
-          <tr  ng-repeat="org in vm.pagedItems" class="hover-area">
+          <tr  ng-repeat="org in vm.filteredItems" class="hover-area">
             <td class="company-logo">
               <a ng-if="org.website" href="{{org.website}}" target="_blank" title="Visit website"><img class="rounded" width="36" height="36" src="{{ org.orgImageURL }}"></a>
               <img ng-if="!org.website" class="rounded border" width="36" height="36" src="{{ org.orgImageURL }}">
             </td>
             <td data-automation-id="holderCompanyName" id="holderCompanyName" class="company-name font-weight-bold text-truncate">{{ org.name }}</td>
             <td class="member-count"><i class="fas fa-users" title="{{ org.members.length }} members"></i> {{ org.members.length }}</td>
-            <td class="swu-status"><i ng-if="vm.hasOrgMetRFQ(org)" title="Sprint With Us Qualified" class="fas fa-certificate"></i></td>
+            <td class="swu-status"><i ng-if="org.hasMetRFQ" title="Sprint With Us Qualified" class="fas fa-certificate"></i></td>
```

Comment:
This is good.  One thing we have to do before putting into production is make sure that all orgs currently in possession of the RFQ have this flag set in prod.  We should test this out in dev first to make sure there are no side-effects.